client/src/app/components/Loginform.js

```diff
@@ -51,14 +74,14 @@ export  class Loginform extends React.Component{
                       <Input  s={12} label="Username"
```

Comment:
what is s? Variable names should be self explanatory.

Reply:
unfortunately that's how materialize allows me to change my values of the label.